manifests/frr_exporter.pp

```diff
@@ -140,30 +143,77 @@
   $all_opts = [$_frr_socket_dir_opt, $_listen_address_opt, $_telemetry_path_opt, $_log_level_opt] + $all_collector_opts
   $options = join($all_opts, ' ')
 
-  prometheus::daemon { $service_name:
-    install_method     => $install_method,
-    version            => $version,
-    download_extension => $download_extension,
-    os                 => $os,
-    arch               => $arch,
-    real_download_url  => $real_download_url,
-    bin_dir            => $bin_dir,
-    notify_service     => $notify_service,
-    package_name       => $package_name,
-    package_ensure     => $package_ensure,
-    manage_user        => $manage_user,
-    service_ensure     => $service_ensure,
-    manage_service     => $manage_service,
-    service_enable     => $service_enable,
-    user               => $user,
-    extra_groups       => $extra_groups,
-    group              => $group,
-    manage_group       => $manage_group,
-    purge_config_dir   => $purge_config_dir,
-    restart_on_change  => $restart_on_change,
-    init_style         => $init_style,
-    service_name       => $service_name,
-    options            => $options,
-    env_vars           => $env_vars,
+  # Extract port from web_listen_address for scrape configuration
```

Comment:
why are you  duplicating the whole prometheus::daemon section here?

Reply:
that was resolved on the latest push